docs/_tutorials/local_development.md

```diff
@@ -1,5 +1,124 @@
+
+![Configuring a Request URL](assets/request-url-config.png "Configuring a Request URL")
+
+## Listening to workspace events
+If you want your app to listen to events like when a reaction is added, when a user mentions your app, or [another Events API event](https://api.slack.com/events), you'll need to enable events for your app.
+
+> The Events API uses the `@slack/events-api` package.
+
+Start by clicking **Event Subscriptions** on the left sidebar. After you toggle the switch, you'll see you need to add a Request URL. This is similar to [adding a Request URL in the interactive components section](#what-is-a-Request-URL). There are two differences though: 
+1. The default path is `/slack/events` instead of `/slack/actions`
+2. The Request URL in this section will need to verify that your endpoint can handle events.
+
+To do this, we've included a command line tool in the `@slack/events-api` package that helps you verify your endpoint.
+
+### Verifying your local endpoint
+If you're using the development proxy as [described above](#using-a-local-request-URL-for-development), you can run the tool from inside your project directory (after the `@slack/events-api` package is installed). You should **only use the command line tool in development**. If your app is up and running, the adapter will automatically respond to challenges.
+
+You can run the command line tool form inside your project directory with the following command:
+
+```
+./node_modules/.bin/slack-verify --secret <signing_secret> [--path=/slack/events] [--port=3000]
+```
+
+You'll need to substitue your own signing secret for `<signing_secret>`. This is found on the **Basic Information** page in the [Create an App section](#create-an-app).
+
+![Signing Secret](assets/signing-secret.png "Signing Secret on Basic Information page")
+
+The path and port values are optional. If your request URL includes a different path, you should specify it with `--path=/my/path/here` (no brackets). Similarly, if your development proxy is forwarding requests to a different port, you should specify it with `--port=8888` (no brackets). If you're using the defaults, you can ignore everything after `<signing_secret>`.
+
+Now that you've verified your endpoint, you can add events.
+
+### Adding event subscriptions
+On the **Event Subscriptions** page, you can add each event your want your app to subscribe to. In the tables below, you may add Workspace events and Bot events. 
+
+Most events require additional scopes (for example, the `reaction_added` event requires the `reactions:read` scope). You can add scopes to your app on the **OAuth & Permissions** page.
+
+Once you've selected all the event types, be sure to Save Changes.
+
+Lastly, if you've added event types that require scopes your app did not previously have, you'll need to reinstall the app into the workspace(s) you'd like Slack to send your app new events. You can reinstall the app from the **Install App** page.
+
+## Next steps
```

Comment:
I think next steps should point you back to events api documentation page (within this site), or interactive messages documentation page (within this site). getting help is listed there.